streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamFlatMapValuesTest.java

```diff
@@ -92,9 +117,9 @@ public void testFlatMapValuesWithKeys() {
 
         stream.flatMapValues(mapper).process(processor);
 
-        driver.setUp(builder);
+        driver = new TopologyTestDriver(builder.build(), props);
         for (final int expectedKey : expectedKeys) {
-            driver.process(topicName, expectedKey, expectedKey);
+            driver.pipeInput(recordFactory.create(topicName, expectedKey, expectedKey, (long) expectedKey));
```

Comment:
Why do we need to pass in `expectedKey` as timestamp?

Reply:
We don't. We just need to pass some long in order to disambiguate the call, because calling `recordFactory.create(String, int, int)` matches two implementations. I'll just call it with `recordFactory.create(topicName, expectedKey, expectedKey, 0L)` to make it clearer.

Reply:
Thanks. Maybe add a comment why we need to pass the timestamp argument?

Reply:
I made the change and added a comment to clarify it, as suggested.